### ISSUE TITLE
ah weirdness

### ISSUE THREAD

**Issue by shawngraham** [edsu/etudier#9]
Not a useful issue title, I know, but...

- was running etudier successfully in a 3.6 environment. Activated the environment today, and though etudier is installed, its as if it doesn't exist;
- for no apparent reason, now recognizes that it exists, but I get error, `caught exception 'data-cid'`

...all of which probably points to me doing something weird, but what?

**Comment by okcgw** [comment]
Same problem here

**Comment by wolfiex** [comment]
replicated.

**Comment by wolfiex** [comment]
Temporary solution by wrapping `from_pub = get_metadata(e)` with a try and except.  Seems to be a problem with the parsing of specific links. 

```
    for e in html.find('#gs_res_ccl_mid .gs_r'):
        
        try:
            from_pub = get_metadata(e)
        except Exception as err:
            print(err)
            continue
            
        yield from_pub, to_pub
```

**Comment by hls1138** [comment]
@wolfiex I tried the "try and except" suggestion without success. I only get a ten nodes .gexf  output file and the script stops with 'data-cid'...

**Comment by wolfiex** [comment]
> @wolfiex I tried the "try and except" suggestion without success. I only get a ten nodes .gexf output file and the script stops with 'data-cid'...

Ah that's a shame, I'm not sure what to suggest as it had fixed it in my instance  - sorry.

**Comment by alonasorochynska** [comment]
Please help me to understand where etudier is saving output files?

**Comment by edsu** [comment]
I think the data-cid problem was fixed in v0.0.8. Sorry for not addressing this issue sooner!
